test/tools/d_do_test.d

```diff
@@ -1534,7 +1534,13 @@ static this()
         outfile.writeln("[RUN_TEST] ", escapeShellCommand(runTest));
         auto runTestProc = std.process.spawnProcess(runTest, stdin, outfile, outfile);
         const exitCode = wait(runTestProc);
-        if (exitCode != 0)
+
+        if (exitCode == 125)
```

Comment:
`exitCode == DISABLED`, no need for magic numbers.

Reply:
That would require importing `dshell_prebuilt` into `d_do_test` which already is quite expensive to compile